src/app/docs.js

```diff
@@ -1 +1,6 @@
-import TextDoc from './components/Text/Text.docs';const DOCS = {}; DOCS["Text"] = TextDoc; export default DOCS;
\ No newline at end of file
+import TextDocs from './components/Text/Text.docs';
+
+const DOCS = {}; 
+DOCS["Text"] = TextDocs; 
+
+export default DOCS;
```

Comment:
Also curious why this file got reformatted when it contains the same contents? This should only ever happen in the event of a new component..